printer_mods/Triano/Ender_3Pro_Switchwire/README.md

```diff
@@ -0,0 +1,92 @@
+#  Ender 3 Pro and V2 to Switchwire Mod
+
+This is a conversion of the Ender 3 Pro and V2 to a Switchwire type coreXZ printer.  My goal was to use most of the Ender 3 components while maintaining the core Voron design look and feel.
+I had a lot of fun doing this and am willing to make any suggested changes.  I have incorporated some changes based on requests and will continue to maintain this as long as there is interest.
+
+![Home](Images/Home.png)
```

Comment:
The above have been resolved. (1) Spaces have been removed from the file names. (2) The redundant file has been removed.  The only difference is the Ender 3 bed has different dimensions in XY and the overall Z is different.  Otherwise, there isn't much difference than the stock Switchwire. (3) The stock STLs removed and the README has been annotated. (4) The STL above print orientation has been fixed.  (5) Additional images added.